src/api_calls.ts

```diff
@@ -1,97 +1,51 @@
-import APIKEYS from "./apiKEYS";
-import axios from 'axios';
+import { APIStrategy } from "./api_strategy";
+import ConfigManager from "./config_manager";
 
 const FormData = require('form-data');
 
-type response = {
-    status: number,
-    body: {
-        type: string,
-        string: string,
-    }
+export type CommonDataModel = {
+    api_name: string,
+    last_modification_date?: number | string,
```

Comment:
Should normalize dates to a particular format. Probably timestamps instead of human readable strings.

Reply:
There is currently no way of configuring the format of a date, but we do plan to work to convert the one returned by VirusTotal.

Reply:
The problem with normalizing dates is that we would need to assume that a date is in a specific format, and so our reasoning (or at least my reasoning) was that we should display the data as is, instead of converting it automatically and possibly giving back incorrect times for an unforeseen format.